starknet_py/tests/e2e/conftest.py

```diff
@@ -137,30 +114,26 @@ def create_rpc_client(run_devnet):
 # pylint: disable=redefined-outer-name
 def create_account_client(pytestconfig, gateway_client):
     net = pytestconfig.getoption("--net")
-    client = None
 
     if net == "testnet":
         key_pair = KeyPair.from_private_key(int(TESTNET_ACCOUNT_PRIVATE_KEY, 0))
-        client = AccountClient(
+        return AccountClient(
             address=TESTNET_ACCOUNT_ADDRESS,
             client=gateway_client,
             key_pair=key_pair,
         )
 
     if net == "integration":
         key_pair = KeyPair.from_private_key(int(INTEGRATION_ACCOUNT_PRIVATE_KEY, 0))
-        client = AccountClient(
+        return AccountClient(
             address=INTEGRATION_ACCOUNT_ADDRESS,
             client=gateway_client,
             key_pair=key_pair,
         )
 
-    if net == "devnet":
-        key_pair = KeyPair.from_private_key(int(DEVNET_ACCOUNT_PRIVATE_KEY, 0))
-        client = AccountClient(
-            address=DEVNET_ACCOUNT_ADDRESS,
-            client=gateway_client,
-            key_pair=key_pair,
-        )
-
-    return client
+    key_pair = KeyPair.from_private_key(int(DEVNET_ACCOUNT_PRIVATE_KEY, 0))
+    return AccountClient(
+        address=DEVNET_ACCOUNT_ADDRESS,
+        client=gateway_client,
+        key_pair=key_pair,
+    )
```

Comment:
Now I've realized: could we replace this lines with helper method that takes `private_key` and `address` and returns an account client? This would reduce duplication in if branches.